### ISSUE TITLE
Make issues on github easier to find

### ISSUE THREAD

**Issue by kaustubh-nair** [publiclab/plots2#5536]
Plots2 has a lot of issues. Most of the times I have to scan through a lot of open issues to find one that I can solve, since a lot of them are already taken. I think it would be a good idea for reviewers to add the contributor as an 'Assignee' so that new issues are easier to find.

Also they can periodically check if the contributor has been inactive and remove them as an 'Assignee' (We can also create a script easily that does this using the Github REST API )

**Comment by gauravano** [comment]
Yes, it's tough to find out issues. How about using a bot for assigning issues? Also, we currently use `claimed` , `assigned` labels for FTOs.

**Comment by gauravano** [comment]
Also, see https://github.com/publiclab/plots2/issues/5237 Thanks!
